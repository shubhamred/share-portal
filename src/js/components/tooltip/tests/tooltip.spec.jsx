import React from 'react';
import { mount, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Tooltip from '../tooltip';

configure({ adapter: new Adapter() });

const props = {
  onChange: jest.fn(),
  value: '',
  children: <p>Test</p>,
  title: 'Test'
};
describe('components/slider', () => {
  it('Renders default options', () => {
    const component = mount(
      <Tooltip {...props} />
    );
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
