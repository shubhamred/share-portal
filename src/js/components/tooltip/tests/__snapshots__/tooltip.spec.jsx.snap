// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/slider Renders default options 1`] = `
<MuiTooltips
  anchorOriginHorizontal="center"
  anchorOriginVertical="bottom"
  onChange={[MockFunction]}
  title="Test"
  transformOriginHorizontal="center"
  transformOriginVertical="top"
  value=""
>
  <div>
    <div
      aria-describedby={null}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      onTouchStart={[Function]}
    >
      <p>
        Test
      </p>
    </div>
    <ForwardRef
      anchorEl={null}
      anchorOrigin={
        Object {
          "horizontal": "center",
          "vertical": "bottom",
        }
      }
      classes={
        Object {
          "paper": "makeStyles-paper-1",
        }
      }
      disableAutoFocus={false}
      disableEnforceFocus={false}
      disableRestoreFocus={true}
      id="demoPopper"
      onClose={[Function]}
      onMouseLeave={[Function]}
      open={false}
      transformOrigin={
        Object {
          "horizontal": "center",
          "vertical": "top",
        }
      }
    >
      <WithStyles(ForwardRef(Popover))
        PaperProps={
          Object {
            "style": Object {
              "pointerEvents": "auto",
            },
          }
        }
        anchorEl={null}
        anchorOrigin={
          Object {
            "horizontal": "center",
            "vertical": "bottom",
          }
        }
        classes={
          Object {
            "paper": "makeStyles-paper-1",
          }
        }
        disableAutoFocus={false}
        disableEnforceFocus={false}
        disableRestoreFocus={true}
        id="demoPopper"
        onClose={[Function]}
        onMouseLeave={[Function]}
        open={false}
        style={
          Object {
            "pointerEvents": "none",
          }
        }
        transformOrigin={
          Object {
            "horizontal": "center",
            "vertical": "top",
          }
        }
      >
        <ForwardRef(Popover)
          PaperProps={
            Object {
              "style": Object {
                "pointerEvents": "auto",
              },
            }
          }
          anchorEl={null}
          anchorOrigin={
            Object {
              "horizontal": "center",
              "vertical": "bottom",
            }
          }
          classes={
            Object {
              "paper": "MuiPopover-paper makeStyles-paper-1",
              "root": "MuiPopover-root",
            }
          }
          disableAutoFocus={false}
          disableEnforceFocus={false}
          disableRestoreFocus={true}
          id="demoPopper"
          onClose={[Function]}
          onMouseLeave={[Function]}
          open={false}
          style={
            Object {
              "pointerEvents": "none",
            }
          }
          transformOrigin={
            Object {
              "horizontal": "center",
              "vertical": "top",
            }
          }
        >
          <ForwardRef(Modal)
            BackdropProps={
              Object {
                "invisible": true,
              }
            }
            className="MuiPopover-root"
            disableAutoFocus={false}
            disableEnforceFocus={false}
            disableRestoreFocus={true}
            id="demoPopper"
            onClose={[Function]}
            onMouseLeave={[Function]}
            open={false}
            style={
              Object {
                "pointerEvents": "none",
              }
            }
          />
        </ForwardRef(Popover)>
      </WithStyles(ForwardRef(Popover))>
    </ForwardRef>
  </div>
</MuiTooltips>
`;
